docs.feldera.com/docs/formats/raw.md

```diff
@@ -30,16 +30,18 @@ This limitation exists because the raw data stream lacks the metadata to differe
 
 In order to ingest raw data:
 
-1. Declare a table with a single column of type `VARBINARY [NOT NUL]` or `VARCHAR [NOT NULL]`.  In the latter case, the connector
+1. Declare a table with a column of type `VARBINARY [NOT NUL]` or `VARCHAR [NOT NULL]`.  In the latter case, the connector
    will parse input data as a UTF-8 string and will fail to parse chunks that are not valid UTF-8.
+   The table can contain additional columns, but all other columns in the table must be either nullable or
```

Comment:
how come you don't need the compiler to check this?
what happen if they aren't?

Reply:
I guess the result is that you get a runtime error when you try to start the pipeline.
Isn't that a bit late?